apps/web/components/Organization/Wrapper/Themes/LandlaeknirTheme/LandlaeknirHeader.css.ts

```diff
@@ -1,27 +1,30 @@
 import { style } from '@vanilla-extract/css'
-import { theme, themeUtils } from '@island.is/island-ui/theme'
+import { themeUtils } from '@island.is/island-ui/theme'
 
 export const headerBg = style({
   height: 385,
   marginTop: -130,
   paddingTop: 130,
-  backgroundRepeat: 'no-repeat',
-  backgroundPositionX: '80%',
-  backgroundPositionY: '500%',
-  borderBottom: `8px solid ${theme.color.blue600}`,
+  backgroundRepeat: 'no-repeat, no-repeat',
+
   ...themeUtils.responsiveStyle({
-    md: {
-      backgroundPositionX: '95%',
+    xs: {
+      backgroundSize: 'cover',
       backgroundImage:
-        "url('https://images.ctfassets.net/8k0h54kbe6bj/2bVBDkDpsHFiOKCM1NgyJD/15c75aeb1c1ac8348e76a24aa2de3e3f/Folk_Saman.png')",
+        'url(https://images.ctfassets.net/8k0h54kbe6bj/1FB32FjyMGC1PfpDfB2Kj1/257dc1108da254359f8988dfef536936/Pattern_PP_Pattern_Circle_Blue_1.png)',
```

Comment:
This new URL is used in 3 places in this file. Maybe consider storing it in a constant variable?